### PR TITLE
Add post money transfer

### PR DESCRIPTION
**Pull request by eweise**
https://trizic.atlassian.net/browse/TFR-6

### CODE REVIEW

**Comment by eweise** [comment]
@johntowell @olmstead @antoinemeyer

File: api/v2/account/examples/bankTransfer.res.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "status": "OK",
+  "message": "OK"
+}
```
Comment (johntowell) [inline api/v2/account/examples/bankTransfer.res.json]: use api/common/examples/generic.res.json

File: api/v2/account/schema/bankTransfer.res.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "message": {
+      "type": "string"
+    }
+  },
+  "required": [
+    "status",
+    "message"
+  ]
+}
+
```
Comment (johntowell) [inline api/v2/account/schema/bankTransfer.res.json]: use api/common/schema/generic.res.json

**Comment by eweise** [comment]
updated

File: api/v2/account/bankTransfer.raml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+post:
+  body:
+    application/json:
+      example: !include examples/bankTransfer.req.json
+      schema: !include schema/bankTransfer.req.json
+  responses:
+    201:
+      body:
+        application/json:
+         schema: !include ../../../api/common/schema/generic.res.json
```
Comment (johntowell) [inline api/v2/account/bankTransfer.raml]: why you so literal? :) All the other ramls are starting at common and not api. Should be consistent?

```
          schema: !include ../../common/schema/generic.res.json
          example: !include ../../common/examples/generic.res.json
```

File: api/v2/account/examples/bankTransfer.res.json (hunk 1 above)
Comment (johntowell) [inline api/v2/account/examples/bankTransfer.res.json]: I meant use common/examples/generic.res.json

**Comment by eweise** [comment]
updated

**Comment by johntowell** [comment]
👍

**Pull request merged**